diff --git a/libsanitizer/sanitizer_common/sanitizer_linux.cc b/libsanitizer/sanitizer_common/sanitizer_linux.cc
index 5b6f18602e7..dffcf9691ff 100644
--- a/libsanitizer/sanitizer_common/sanitizer_linux.cc
+++ b/libsanitizer/sanitizer_common/sanitizer_linux.cc
@@ -1260,9 +1260,9 @@ struct __sanitizer_esr_context {
   uint64_t esr;
 };
 
-static bool Aarch64GetESR(ucontext_t *ucontext, u64 *esr) {
+static bool Aarch64GetESR(ucontext_t *uContext, u64 *esr) {
   static const u32 kEsrMagic = 0x45535201;
-  u8 *aux = ucontext->uc_mcontext.__reserved;
+  u8 *aux = uContext->uc_mcontext.__reserved;
   while (true) {
     _aarch64_ctx *ctx = (_aarch64_ctx *)aux;
     if (ctx->size == 0) break;
@@ -1277,18 +1277,18 @@ static bool Aarch64GetESR(ucontext_t *ucontext, u64 *esr) {
 #endif
 
 SignalContext::WriteFlag SignalContext::GetWriteFlag(void *context) {
-  ucontext_t *ucontext = (ucontext_t *)context;
+  ucontext_t *uContext = (ucontext_t *)context;
 #if defined(__x86_64__) || defined(__i386__)
   static const uptr PF_WRITE = 1U << 1;
 #if SANITIZER_FREEBSD
-  uptr err = ucontext->uc_mcontext.mc_err;
+  uptr err = uContext->uc_mcontext.mc_err;
 #else
-  uptr err = ucontext->uc_mcontext.gregs[REG_ERR];
+  uptr err = uContext->uc_mcontext.gregs[REG_ERR];
 #endif
   return err & PF_WRITE ? WRITE : READ;
 #elif defined(__arm__)
   static const uptr FSR_WRITE = 1U << 11;
-  uptr fsr = ucontext->uc_mcontext.error_code;
+  uptr fsr = uContext->uc_mcontext.error_code;
   return fsr & FSR_WRITE ? WRITE : READ;
 #elif defined(__aarch64__)
   static const u64 ESR_ELx_WNR = 1U << 6;
@@ -1303,80 +1303,80 @@ SignalContext::WriteFlag SignalContext::GetWriteFlag(void *context) {
 
 void GetPcSpBp(void *context, uptr *pc, uptr *sp, uptr *bp) {
 #if defined(__arm__)
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.arm_pc;
-  *bp = ucontext->uc_mcontext.arm_fp;
-  *sp = ucontext->uc_mcontext.arm_sp;
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.arm_pc;
+  *bp = uContext->uc_mcontext.arm_fp;
+  *sp = uContext->uc_mcontext.arm_sp;
 #elif defined(__aarch64__)
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.pc;
-  *bp = ucontext->uc_mcontext.regs[29];
-  *sp = ucontext->uc_mcontext.sp;
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.pc;
+  *bp = uContext->uc_mcontext.regs[29];
+  *sp = uContext->uc_mcontext.sp;
 #elif defined(__hppa__)
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.sc_iaoq[0];
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.sc_iaoq[0];
   /* GCC uses %r3 whenever a frame pointer is needed.  */
-  *bp = ucontext->uc_mcontext.sc_gr[3];
-  *sp = ucontext->uc_mcontext.sc_gr[30];
+  *bp = uContext->uc_mcontext.sc_gr[3];
+  *sp = uContext->uc_mcontext.sc_gr[30];
 #elif defined(__x86_64__)
 # if SANITIZER_FREEBSD
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.mc_rip;
-  *bp = ucontext->uc_mcontext.mc_rbp;
-  *sp = ucontext->uc_mcontext.mc_rsp;
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.mc_rip;
+  *bp = uContext->uc_mcontext.mc_rbp;
+  *sp = uContext->uc_mcontext.mc_rsp;
 # else
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.gregs[REG_RIP];
-  *bp = ucontext->uc_mcontext.gregs[REG_RBP];
-  *sp = ucontext->uc_mcontext.gregs[REG_RSP];
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.gregs[REG_RIP];
+  *bp = uContext->uc_mcontext.gregs[REG_RBP];
+  *sp = uContext->uc_mcontext.gregs[REG_RSP];
 # endif
 #elif defined(__i386__)
 # if SANITIZER_FREEBSD
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.mc_eip;
-  *bp = ucontext->uc_mcontext.mc_ebp;
-  *sp = ucontext->uc_mcontext.mc_esp;
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.mc_eip;
+  *bp = uContext->uc_mcontext.mc_ebp;
+  *sp = uContext->uc_mcontext.mc_esp;
 # else
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.gregs[REG_EIP];
-  *bp = ucontext->uc_mcontext.gregs[REG_EBP];
-  *sp = ucontext->uc_mcontext.gregs[REG_ESP];
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.gregs[REG_EIP];
+  *bp = uContext->uc_mcontext.gregs[REG_EBP];
+  *sp = uContext->uc_mcontext.gregs[REG_ESP];
 # endif
 #elif defined(__powerpc__) || defined(__powerpc64__)
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.regs->nip;
-  *sp = ucontext->uc_mcontext.regs->gpr[PT_R1];
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.regs->nip;
+  *sp = uContext->uc_mcontext.regs->gpr[PT_R1];
   // The powerpc{,64}-linux ABIs do not specify r31 as the frame
   // pointer, but GCC always uses r31 when we need a frame pointer.
-  *bp = ucontext->uc_mcontext.regs->gpr[PT_R31];
+  *bp = uContext->uc_mcontext.regs->gpr[PT_R31];
 #elif defined(__sparc__)
-  ucontext_t *ucontext = (ucontext_t*)context;
+  ucontext_t *uContext = (ucontext_t*)context;
   uptr *stk_ptr;
 # if defined (__arch64__)
-  *pc = ucontext->uc_mcontext.mc_gregs[MC_PC];
-  *sp = ucontext->uc_mcontext.mc_gregs[MC_O6];
+  *pc = uContext->uc_mcontext.mc_gregs[MC_PC];
+  *sp = uContext->uc_mcontext.mc_gregs[MC_O6];
   stk_ptr = (uptr *) (*sp + 2047);
   *bp = stk_ptr[15];
 # else
-  *pc = ucontext->uc_mcontext.gregs[REG_PC];
-  *sp = ucontext->uc_mcontext.gregs[REG_O6];
+  *pc = uContext->uc_mcontext.gregs[REG_PC];
+  *sp = uContext->uc_mcontext.gregs[REG_O6];
   stk_ptr = (uptr *) *sp;
   *bp = stk_ptr[15];
 # endif
 #elif defined(__mips__)
-  ucontext_t *ucontext = (ucontext_t*)context;
-  *pc = ucontext->uc_mcontext.pc;
-  *bp = ucontext->uc_mcontext.gregs[30];
-  *sp = ucontext->uc_mcontext.gregs[29];
+  ucontext_t *uContext = (ucontext_t*)context;
+  *pc = uContext->uc_mcontext.pc;
+  *bp = uContext->uc_mcontext.gregs[30];
+  *sp = uContext->uc_mcontext.gregs[29];
 #elif defined(__s390__)
-  ucontext_t *ucontext = (ucontext_t*)context;
+  ucontext_t *uContext = (ucontext_t*)context;
 # if defined(__s390x__)
-  *pc = ucontext->uc_mcontext.psw.addr;
+  *pc = uContext->uc_mcontext.psw.addr;
 # else
-  *pc = ucontext->uc_mcontext.psw.addr & 0x7fffffff;
+  *pc = uContext->uc_mcontext.psw.addr & 0x7fffffff;
 # endif
-  *bp = ucontext->uc_mcontext.gregs[11];
-  *sp = ucontext->uc_mcontext.gregs[15];
+  *bp = uContext->uc_mcontext.gregs[11];
+  *sp = uContext->uc_mcontext.gregs[15];
 #else
 # error "Unsupported arch"
 #endif
diff --git a/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc b/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc
index 31a5e697eae..e482e12ca3b 100644
--- a/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc
+++ b/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc
@@ -154,7 +154,7 @@ typedef struct user_fpregs elf_fpregset_t;
 # include <sys/procfs.h>
 #endif
 #include <sys/user.h>
-#include <sys/ustat.h>
+//#include <sys/ustat.h>
 #include <linux/cyclades.h>
 #include <linux/if_eql.h>
 #include <linux/if_plip.h>
@@ -247,7 +247,7 @@ namespace __sanitizer {
 #endif // SANITIZER_LINUX || SANITIZER_FREEBSD
 
 #if SANITIZER_LINUX && !SANITIZER_ANDROID
-  unsigned struct_ustat_sz = sizeof(struct ustat);
+  //unsigned struct_ustat_sz = sizeof(struct ustat);
   unsigned struct_rlimit64_sz = sizeof(struct rlimit64);
   unsigned struct_statvfs64_sz = sizeof(struct statvfs64);
 #endif // SANITIZER_LINUX && !SANITIZER_ANDROID

